docs/operations/images.md

```diff
@@ -1,167 +1,235 @@
 # Images
 
-Changing the image for an instance group
+The default images for Kops clusters are based on Debian 9 (Stretch). These images are built by the [kubernetes-sigs/image-builder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/kube-deploy/imagebuilder) project for each Kubernetes release and are available for any other use under the `kope.io` account.
 
-You can choose a different AMI for an instance group.
+You can choose a different image for an instance group by editing it with `kops edit ig nodes`. You should see an `image` field in one of the following formats:
 
-If you `kops edit ig nodes`, you should see an `image` member of the spec.
+* `ami-abcdef` - specifies an AMI by id directly
+* `<owner>/<name>` specifies an AMI by its owner and name properties
+* `<alias>/<name>` specifies an AMI by its owner alias and name properties
 
-Various syntaxes are available:
+Using the AMI id is precise, but ids vary by region. It is often more convenient to use the `<owner/alias>/<name>`, if equivalent images have been copied to various regions with the same name.
 
-* `ami-abcdef` specifies an AMI by id directly.
-* `<owner>/<name>` specifies an AMI by its owner and Name properties
+Kops supports owner aliases for the official accounts of supported distros:
 
-The ami spec is precise, but AMIs vary by region.  So it is often more convenient to use the `<owner>/<name>`
-specifier, if equivalent images have been copied to various regions with the same name.
+* `kope.io` => `383156758163`
+* `amazon` => `137112412989`
+* `centos` => `679593333241`
+* `debian9` => `379101102735`
+* `debian10` => `136693071363`
+* `flatcar` => `075585003325`
+* `redhat` => `309956199498`
+* `ubuntu` => `099720109477`
+
+For example, to use Ubuntu 20.04, you could specify:
+
+```yaml
+image: ami-00579fbb15b954340
+```
 
-For example, to use Ubuntu 16.04, you could specify:
+or
 
-`image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-xenial-16.04-amd64-server-20180405`
+```yaml
+image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-arm64-server-20200423
```

Comment:
maybe we use amd64 examples since they're more popular than arm at the moment? same with below :)

Reply:
I may have tunnel vision from all the ARM stuff these last few weeks 😄